test-server/test-echo.c

```diff
@@ -208,7 +211,10 @@ int main(int argc, char **argv)
 	int disallow_selfsigned = 0;
 #endif
 
+#ifdef WIN32
```

Comment:
I introduced this to avoid the following warning:

w:/libwebsockets/test-server/test-echo.c: In function 'main':
w:/libwebsockets/test-server/test-echo.c:214:6: warning: unused variable 'debug_level' [-Wunused-variable]
  int debug_level = 7;
      ^
Wrt #ifdef/#ifndef WIN32/_WIN32: I've no problems to change that as suggested - but note that the same pattern is also used in line 275, 323, 332, and 431.